code401challenges/src/main/java/code401challenges/linkedlist/LinkedList.java

```diff
@@ -69,19 +125,20 @@ private Node find(String target) {
     // Define a method called toString (or __str__ in Python) which takes in no arguments
     // and returns a string representing all the values in the Linked List, formatted as:
     // "{ a } -> { b } -> { c } -> NULL"
+
+    // update this to match with new toString() method
     public String toString() {
         Node position = head;
         String LinkedListString = "";
```

Comment:
Instead of defining your LinkedListString as a String, you could do a StringBuilder.
`StringBuilder LinkedListString = new StringBuilder();`